HeterogeneousCore/CUDAUtilities/BuildFile.xml

```diff
@@ -0,0 +1,2 @@
+<use   name="cuda"/>
```

Comment:
I would assume this file is not needed as there is no `src` directory to compile.

Reply:
True, but `GPUSimpleVector.h` does include CUDA files, so I thought better to add it.

Reply:
Ok. (and eventually it is likely needed anyway)